### ISSUE TITLE
Remove TurboJPEG from Cantaloupe (disable is not enough) to allow PDF Y coordinate level tiling

### ISSUE THREAD

**Issue by DiegoPino** [esmero/archipelago-docker-images#60]
See https://github.com/cantaloupe-project/cantaloupe/issues/482

For some reason (explained there) when turbojpeg is around PDF tiling on Y coordinate is dead. Unmantained software, was fixed in 5.02, broke again in 5.04, and of course broken on 6.x

**Comment by DiegoPino** [comment]
See https://github.com/cantaloupe-project/cantaloupe/issues/482#issuecomment-1674917947

I will fork and point our docker to a new branch instead

**Comment by patdunlavey** [comment]
I did not try that. I will give that a test!

**Comment by patdunlavey** [comment]
Thanks for pointing out what I should have noticed would be an obvious change needed in cantaloupe.properties when [libjpeg-turbo](https://sourceforge.net/projects/libjpeg-turbo) is removed. But in further testing, what I didn't expect, is that having `processor.ManualSelectionStrategy.jpg = TurboJpegProcessor`, when using the [6.0.0-noturbojpeg-multiarch](https://hub.docker.com/layers/esmero/cantaloupe-s3/6.0.0-noturbojpeg-multiarch/images/sha256-6ab80b37755c4cdb97b093fe5283aaa47c73aa19e8421235ac841ff3c3f7bf67?context=explore) image, seems to work just fine!
Perhaps cantaloupe's TurboJpegProcessor fails back to use Java2D automatically if TurboJpeg is not found?

**Comment by patdunlavey** [comment]
I should clarify/emphasize that the jp2 files that fail are grayscale. It seems that color jp2s work fine with libjpeg-turbo installed.

**Comment by DiegoPino** [comment]
@patdunlavey yes Derek explained your issue with B&W JP2s on slack. Only real way of getting TurboJPEG out of the way is removing the library as I did in https://hub.docker.com/layers/esmero/cantaloupe-s3/6.0.0-noturbojpeg-multiarch/images/sha256-6ab80b37755c4cdb97b093fe5283aaa47c73aa19e8421235ac841ff3c3f7bf67?context=explore

Probably the issue is on the writer (which as you detected will not be used if the library is not present)... 

https://github.com/cantaloupe-project/cantaloupe/blob/21057cf72071840e2a984450bf8845c6e6490ec9/src/main/java/edu/illinois/library/cantaloupe/processor/codec/jpeg/TurboJPEGImageWriter.java#L18

but also there is this very old ISSUE open which matches the issue you are seeing with greyscale...
https://github.com/cantaloupe-project/cantaloupe/issues/190

There is a pull that addressed one of the problems but not the actual intermediate BMP issue with low bit colors. You all sure Kakadu did not work for this?

**Comment by DiegoPino** [comment]
Wonder also if you have write permissions to the temporary file system...
See this:

https://github.com/cantaloupe-project/cantaloupe/blob/21057cf72071840e2a984450bf8845c6e6490ec9/src/main/java/edu/illinois/library/cantaloupe/processor/OpenJpegProcessor.java#L395-L405
